SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1609,9 +1620,11 @@ void USpatialReceiver::OnCreateEntityResponse(const Worker_CreateEntityResponseO
 	{
 		Channel->OnCreateEntityResponse(Op);
 	}
-	else
+	else if (Channel.IsStale())
```

Comment:
What happens if it's not valid and not stale?

Reply:
The it shouldn't print any of these. This is here because we treat channels differently from other delegates. If you have neither (and never had a channel) then it just reverts to the verbose logging above.